packages/@lwc/engine-server/src/utils/validate-style-text-contents.ts

```diff
@@ -5,44 +5,46 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 
-import * as parse5 from 'parse5';
-import { DocumentFragment } from '@parse5/tools';
-
-function isSingleStyleNodeContainingSingleTextNode(node: DocumentFragment) {
-    if (node.childNodes.length !== 1) {
-        return false;
-    }
-
-    const style = node.childNodes[0];
-
-    if (style.nodeName !== 'style' || style.childNodes.length !== 1) {
-        return false;
-    }
-
-    const textNode = style.childNodes[0];
-
-    return textNode.nodeName === '#text';
-}
+/**
+ * Per the HTML spec on restrictions for "raw text elements" like `<style>`:
+ * <blockquote>
+ * The text in raw text and escapable raw text elements must not contain any occurrences of the string
+ * "</" (U+003C LESS-THAN SIGN, U+002F SOLIDUS) followed by characters that case-insensitively match the tag name of
+ * the element followed by one of:
+ * - U+0009 CHARACTER TABULATION (tab)
+ * - U+000A LINE FEED (LF)
+ * - U+000C FORM FEED (FF)
+ * - U+000D CARRIAGE RETURN (CR)
+ * - U+0020 SPACE
+ * - U+003E GREATER-THAN SIGN (>), or
+ * - U+002F SOLIDUS (/)
+ * </blockquote>
```

Comment:
_(This doesn't really matter because the constant isn't exported, so people are just going to read the code comment, rather than the IDE tooltip.)_

---

VS Code doesn't render the contents of the `<blockquote>`:

> ![screenshot of VS Code tooltip with `<blockquote>` contents missing](https://github.com/salesforce/lwc/assets/62956339/6e638de3-5617-4450-b77f-2ae3df6e59c7)

```suggestion
 *
 * The text in raw text and escapable raw text elements must not contain any occurrences of the string
 * "</" (U+003C LESS-THAN SIGN, U+002F SOLIDUS) followed by characters that case-insensitively match the tag name of
 * the element followed by one of:
 * - U+0009 CHARACTER TABULATION (tab)
 * - U+000A LINE FEED (LF)
 * - U+000C FORM FEED (FF)
 * - U+000D CARRIAGE RETURN (CR)
 * - U+0020 SPACE
 * - U+003E GREATER-THAN SIGN (>), or
 * - U+002F SOLIDUS (/)
```

Removing the `<blockquote>` renders it thusly:

> ![screenshot of VS Code tooltip, showing the contents with `<blockquote>` removed](https://github.com/salesforce/lwc/assets/62956339/b639462a-7e0b-4344-8020-3f16e5b1f9a0)

```suggestion
 *
 * > The text in raw text and escapable raw text elements must not contain any occurrences of the string
 * > "</" (U+003C LESS-THAN SIGN, U+002F SOLIDUS) followed by characters that case-insensitively match the tag name of
 * > the element followed by one of:
 * > - U+0009 CHARACTER TABULATION (tab)
 * > - U+000A LINE FEED (LF)
 * > - U+000C FORM FEED (FF)
 * > - U+000D CARRIAGE RETURN (CR)
 * > - U+0020 SPACE
 * > - U+003E GREATER-THAN SIGN (>), or
 * > - U+002F SOLIDUS (/)
```

Converting to markdown blockquotes renders it thusly:

> ![screenshot of VS Code tooltip, showing the contents when using markdown `>` blockquotes](https://github.com/salesforce/lwc/assets/62956339/5011a367-e49b-4faa-9b7e-907a87ea03d5)

🤷